aea/agent.py

```diff
@@ -191,7 +191,12 @@ def start(self) -> None:
 
         :return: None
         """
-        self.runtime.start()
+        was_started = self.runtime.start()
+
+        if was_started:
+            self.runtime.wait_completed(sync=True)
+        else:
+            raise ValueError("Failed to start runtime! Was it already started?")
```

Comment:
Should this maybe be an `AEAException` instead of a `ValueError`?

Reply:
nice point